.github/workflows/package.yaml

```diff
@@ -64,4 +64,4 @@ jobs:
             -H "Authorization: token ${{ secrets.ANALYSIS_SERVER_GITHUB_TOKEN }}" \
             -H "Accept: application/vnd.github.v3+json" \
             https://api.github.com/repos/populationgenomics/analysis-runner/actions/workflows/6364059/dispatches \
-            -d '{"ref": "main", "inputs": {"hail_version": "'$version'"}}'
+            -d '{"ref": "main", "inputs": {"hail_version": "'$version'", "hail_sha": "${{ github.sha }}"}}'
```

Comment:
Flagging this comment as related: https://github.com/populationgenomics/analysis-runner/pull/349#discussion_r826409349